db/schema.rb

```diff
@@ -621,6 +623,8 @@
     t.datetime "updated_at", precision: nil, null: false
     t.datetime "bounced_at", precision: nil
     t.datetime "reminder_sent_at", precision: nil
+    t.datetime "expiration_notice_sent_at", precision: nil
+    t.index ["expiration_notice_sent_at"], name: "index_usps_confirmation_codes_on_expiration_notice_sent_at"
```

Comment:
Since you're not notifying in this PR, do you still want to add the column for that?

Reply:
Oooh good catch

Reply:
removed in [28961da](https://github.com/18F/identity-idp/pull/9545/commits/28961da05381381a6aeca2429ac1330462500982)

Reply:
Looks like this got added back in c032fda